testschematic/tests.go

```diff
@@ -143,45 +146,57 @@ func (options *TestSchematicOptions) RunSchematicTest() error {
 		}
 	}
 
-	// ------ DESTROY ------
-	// only run destroy if we had potentially created resources
-	if svc.TerraformResourcesCreated {
```

Comment:
If you are moving the resource destroy from this block, you need to still replace it with your new TestTearDown() or similar.

Keep in mind thought that in this test case there are TWO different destroy that take place:
1. The destroy of the applied resources
2. The delete of the workspace

The block removed here is covering item 1, the destroy of the applied resources, NOT the deletion of the workspace.

Perhaps a short meeting or slack conversation can explain this better.